client/blindai/client.py

```diff
@@ -509,13 +520,13 @@ def upload_model(
 
         return ret
 
-    def run_model(self, data_list: List[Any], sign: bool = False) -> RunModelResponse:
+    def run_model(self, data_list: Union[List[List[Any]], List[Any]], sign: bool = False) -> RunModelResponse:
```

Comment:
Same question than above regarding having models with just one input.

Reply:
Yeah, I believe you can just pass a single list similar to the current API and it will work fine.